npm/templates/deploy.py

```diff
@@ -42,10 +42,11 @@
 }
 npm_registry = npm_repositories[repo_type]
 
-npm_username, npm_password, npm_email = (
+npm_username, npm_password, npm_email, npm_token = (
```

Comment:
I didn't mention it in our discussion, but we should rewrite `deploy.py` in Kotlin. Remember that we're phasing out Python from our build systems.